### PR TITLE
Added clarification on where to mention Acceptance Criteria

### PR DESCRIPTION
**Pull request by nooodle-soup**
Added where to include the Acceptance Criteria to avoid confusion.

----
:books: Documentation preview :books:: https://ploomber-contributing--85.org.readthedocs.build/en/85/

### CODE REVIEW

File: contributing/process-outline.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ interested in working on.
 ## 2. Describe the Acceptance Criteria
 
 - Before starting to work on the issue, describe the acceptance criteria (AC). 
+- Write the AC in the issue when you create one.
```
Comment (edublancas) [inline contributing/process-outline.md:21]: let's put it in the same bullet point (essentially merge this new one with the one above)

**Review by edublancas — CHANGES_REQUESTED** [review]

File: contributing/process-outline.md (hunk 1 above)
Comment (nooodle-soup) [reply contributing/process-outline.md]: Done

File: contributing/process-outline.md [hunk 2]
```diff
@@ -17,7 +17,8 @@ interested in working on.
 
 ## 2. Describe the Acceptance Criteria
 
-- Before starting to work on the issue, describe the acceptance criteria (AC). 
+- Before starting to work on the issue, decide the acceptance criteria (AC)
```
Comment (edublancas) [inline contributing/process-outline.md:20]:
```suggestion
- Before starting to work on the issue, describe the acceptance criteria (AC)
```

**Review by edublancas — CHANGES_REQUESTED** [review]

File: contributing/process-outline.md (hunk 2 above)
Comment (nooodle-soup) [reply contributing/process-outline.md]: Done

**Review by edublancas — APPROVED** [review]

**Pull request merged**